### DISCUSSION TITLE
continue upgrade remaining packages if one builds fails?

### DISCUSSION THREAD

**Discussion by SKNo** [microsoft/vcpkg#20163] [Q&A] [answered]
Hi
I was wondering, is it possible to continue  upgrade if one of packages in the list fails to build?
example 
`vcpkg upgrade --no-dry-run `
aims at building 40 packages of different triplets. If the build process of package 34/40 fails, all build process stops and the remaining packages (35 to 40) are already deleted,  Am I missing something ?

**Comment by SKNo** [comment, marked as answer]
Find the answer
[https://github.com/microsoft/vcpkg/discussions/17922]

**Comment by autoantwort** [comment]
There is an option `--keep-going`
